src/Core/IdentityServer/ApiResources.cs

```diff
@@ -20,7 +20,8 @@ public static IEnumerable<ApiResource> GetApiResources()
                     "orgowner",
                     "orgadmin",
                     "orgmanager",
-                    "orguser"
+                    "orguser",
+                    "orgcustom"
```

Comment:
Trailing comma at the end of the list here please (same reason as the enums above).